tests/functional/monitoring/prometheus/alerts/test_alert_mds_cpu_high_usage.py

```diff
@@ -108,4 +139,5 @@ def test_alert_triggered(self, run_file_creator_io_with_cephfs, threading_lock):
             "File creation IO started in the background."
             " Script will look for MDSCPUUsageHigh  alert"
         )
+
```

Comment:
Please add description that this test is covering only vertical scaling and also why horizontal scaling is not covered in automation.